app/presenters/tree_builder_ops_rbac_features.rb

```diff
@@ -0,0 +1,178 @@
+  has_kids_for Menu::Item,        [:x_get_tree_item_kids]
+  has_kids_for MiqProductFeature, [:x_get_tree_feature_kids]
+
+  attr_reader :role, :features, :editable, :root_counter
+
+  def initialize(name, type, sandbox, build, role:, editable: false)
+    @role     = role
+    @editable = editable
+    @features = @role.miq_product_features.order(:identifier).pluck(:identifier)
+
+    @root_counter = []
+
+    # Make sure tree_state doesn't hold on to old data between requests
+    TreeState.new(sandbox).remove_tree(name)
+
+    super(name, type, sandbox, build)
+  end
+
+  private
+
+  def filtered_menus
+    menus = Menu::Manager.map do |section|
+      next if section.id == :cons && !Settings.product.consumption
+      next if section.name.nil?
+      next unless Vmdb::PermissionStores.instance.can?(section.id)
+
+      section
+    end
+
+    menus
+  end
+
+  def set_locals_for_render
+    locals = {
+      :checkboxes   => true,
+      :three_checks => true,
+      :check_url    => "/ops/rbac_role_field_changed/",
+      :onclick      => nil
+    }
+
+    if editable
+      locals[:oncheck] = "miqOnCheckHandler"
+    end
+
+    super.merge!(locals)
+  end
+
+  def x_get_tree_roots(count_only = false, _options)
+    top_nodes = filtered_menus
+    top_nodes << MiqProductFeature.find_by_identifier("all_vm_rules")
+
+    count_only_or_objects(count_only, top_nodes)
+  end
+
+  def x_get_tree_hash_kids(parent, count_only = false)
+    count_only_or_objects(count_only, parent[:data][:kids])
+  end
+
+  def x_get_tree_section_kids(parent, count_only = false)
+    kids = parent.items.reject do |item|
+      if item.kind_of?(Menu::Item)
+        item.feature.nil? || !MiqProductFeature.feature_exists?(item.feature)
+      end
+    end
+
+    count_only_or_objects(count_only, kids)
+  end
+
+  # FIXME: This is inefficient, but done to use MiqProductFeature objects rather than hashes
+  def x_get_tree_item_kids(parent, count_only = false)
+    parent_feature = all_features.find_by(:identifier => parent.feature)
+
+    if parent_feature
+      details = MiqProductFeature.features[parent.feature]
+
+      kids = parent_feature.children.where(:hidden => [false, nil]).sort_by do |c|
+        details[:children].index(c.identifier)
+      end
+
+      count_only_or_objects(count_only, kids)
+    else
+      []
+    end
+  end
+
+  def x_get_tree_feature_kids(parent, count_only = false)
+    count_only_or_objects(count_only, parent.children)
+  end
+
+  def tree_init_options(_tree_name)
+    {
+      :lazy           => false,
+      :add_root       => true,
+      :role           => role,
+      :features       => features,
+      :editable       => editable,
+      :node_id_prefix => node_id_prefix
+    }
+  end
+
+  def root_options
+    root_node
+  end
+
+  def root_node
+    @root_node ||= {
+      :key         => "#{node_id_prefix}__#{root_feature}",
+      :image       => "100/feature_node.png",
```

Comment:
This should be 

```
 :icon        => 'pficon pficon-folder-close',
```

(It has since been updated in master, I guess you missed in when rebasing.)